src/apps/HeyGPT/HeyGPT.Core/Services/OpenAIService.cs

```diff
@@ -74,10 +74,10 @@ public async Task<ChatCompletionResponse> SendPrompt(ChatCharacterDetails chatCh
 
         var chatCompletionsOptions = new ChatCompletionsOptions
         {
-            DeploymentName = DefaultModelVersion
+            DeploymentName = DefaultModelVersion,
+            MaxTokens = 1000
```

Comment:
Had a wild 6000 token response, so we are hard coding this for now... We really need to add options support for all these parameters so they can be changed in configuration files.